html/semantics/index.md

```diff
@@ -0,0 +1,165 @@
+---
+title: "Что такое семантика и почему нельзя всё верстать на div'ах?"
+description: "Узнайте как работает семантическая вёрстка и зачем она нужна"
+authors:
+  - punkmachine
+related:
+  - a11y/chto-takoe-a11y
+  - a11y/a11y-html
+  - html/seo-for-beginners
+tags:
+  - article
+---
+
+## Кратко
+
+Семантическая вёрстка — особый подход к написанию html разметки страниц. При этом подходе разработчик опирается не на содержание сайта, а на смысловое предназначение каждого блока и логическую структуру html страницы.
+
+## Семантические теги
+
+В современном стандарте HTML существует большое количество семантических тегов под разные логические блоки веб-документа. Например:
+
+- [`<article>`](/html/article/) — тег, использующийся для создания самостоятельных блоков контента. Содержимое тега может быть карточкой статьи в блоге, карточкой товара, рекламным баннером и любым другим самодостаточным логическим блоком.
```

Comment:
Я вот отсюда https://html.spec.whatwg.org/dev/sections.html#the-article-element пытаюсь осмыслить спеку. Местами действительно неоднозначно. Но основной посыл на сильном логическом перекосе, а не физическом (шапка, подва, боковушка). И отдельно ещё тема про Outline - как строится смысловая структура документа на основе секций и заголовков. Хотите вместе попишем?

PS. В доке есть статьи про все теги (все нужно тоже править, пока только до хедера руки дошли), а эта статья должна объединить по идее всё в единое понимание семантики

Reply:
Пропустил этот комментарий из-за того, что гитхаб его решил спрятать под кнопку разворачивающую комменты. 

По идее, если верить буквально первому абзацу спеки, то я вроде бы правильно изложил:
> The [article](https://html.spec.whatwg.org/dev/sections.html#the-article-element) element [represents](https://html.spec.whatwg.org/dev/dom.html#represents) a complete, or self-contained, composition in a document, page, application, or site and that is, in principle, independently distributable or reusable, e.g. in syndication. This could be a forum post, a magazine or newspaper article, a blog entry, a user-submitted comment, an interactive widget or gadget, or any other independent item of content.

Т.е. это любой самостоятельный компонент\блок на странице.

Касаемо вопроса:
> И отдельно ещё тема про Outline - как строится смысловая структура документа на основе секций и заголовков. Хотите вместе попишем?

Я не уверен, что статья не затянется с моей стороны на полгода, как эта, поэтому пока их брать не буду точно😅.